providers/logzio/alerts.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018 The Terraformer Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logzio
+
+import (
+	"log"
+	"strconv"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraform_utils"
+
+	"github.com/jonboydell/logzio_client/alerts"
+)
+
+type AlertsGenerator struct {
+	LogzioService
+}
+
+// Generate Terraform Resources from Logzio API,
+func (g *AlertsGenerator) InitResources() error {
+	var client *alerts.AlertsClient
+	client, _ = alerts.New(g.Args["token"].(string), g.Args["baseURL"].(string))
+
+	alerts, err := client.ListAlerts()
+	if err != nil {
+		log.Println(err)
```

Comment:
you don't need log.println here. Error return from func and printed in main flow

Reply:
Fixed.